src/common/lists.js

```diff
@@ -14,8 +14,11 @@ export class List {
 
   add(description, completed) {
     if (Array.isArray(this.list)) {
-      const index = this.list.length;
+      const storedTodoList = JSON.parse(localStorage.getItem('toDolist')) ?? [];
```

Comment:
- [ ] In this file you have linter disabled - please, consider removing it. Let the linter makes its direct job - find the bugs in the early stages of development 👍

Reply:
Corrected the same. As there is 2 class is declared, so 1 class is disabled.